src/executables/server.c

```diff
@@ -439,6 +439,7 @@ main (int argc, char **argv)
 	fflush (stderr);
       }
   }
+  er_final (ER_ALL_FINAL);
```

Comment:
`er_final` should be called in the same scope that `er_init` is called.
```suggestion
    er_final (ER_ALL_FINAL);
  }
```

Reply:
Moved it now.